oml/metrics/accumulation.py

```diff
@@ -27,12 +28,16 @@ def __init__(self, keys_to_accumulate: Sequence[str]):
         self._collected_samples = 0
         self._storage: TStorage = dict()
 
+        self.indices_key = "element_indices"  # internal key to keep track of elements order if provided
```

Comment:
done (the rest private fields have 1 underscores, so I've also added one)

Reply:
I told about value, like `"__element_indices"`. Because user will see it even if he didn't asked it to collect in `key_to_accumulate`.
Underscore for attribute is also ok for consistency

Reply:
oh, okay, did it